v2/common/src/main/java/com/google/cloud/teleport/v2/utils/BeamSchemaUtils.java

```diff
@@ -100,17 +100,27 @@ private static Schema fromJson(JsonParser jsonParser) throws IOException, Schema
   private static List<Field> getFieldsfromJsonNode(JsonNode jsonNode)
       throws SchemaParseException {
     List<Field> fields = new LinkedList<>();
+    Field field;
```

Comment:
let's move that declaration inside for, because it no need outside it scope.
`Field field = nullable ? Field.nullable(name, stringToFieldType(type))
          : Field.of(name, stringToFieldType(type));`

In any way it will be create new instance of Field